src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -361,79 +362,85 @@ function AvatarCropModal(props) {
             type={CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED}
             onModalHide={resetState}
         >
-            {props.isSmallScreenWidth && <HeaderGap />}
-            <HeaderWithBackButton
-                title={props.translate('avatarCropModal.title')}
-                onBackButtonPress={props.onClose}
-            />
-            <Text style={[styles.mh5]}>{props.translate('avatarCropModal.description')}</Text>
-            <GestureHandlerRootView
-                onLayout={initializeImageContainer}
-                style={[styles.alignSelfStretch, styles.m5, styles.flex1, styles.alignItemsCenter]}
+            <ScreenWrapper
+                style={[styles.modalWrapper]}
```

Comment:
```suggestion
                style={[styles.pb0]}
```

Why do we need this style?